test/fixtures/state-saved-v0.js

```diff
@@ -1169,10 +1169,10 @@ mergedLayer2.config = {
   sizeField: {
     name: 'int_range',
     format: '',
-    id: 'int_range',
     type: 'integer',
-    tableFieldIndex: 10,
-    analyzerType: 'INT'
+    fieldIdx: 9,
+    analyzerType: 'INT',
+    valueAccessor: values => values[0]
```

Comment:
the test logic only validates the presence of the valueAccessor. it's part of test field. It's really difficult to compare functions and that's why we only validate the presence and the validity of the field, e.g not null.
I can definitely update to ```valueAccessor: values => values[9]``` but it will not make a difference.